java/java.source.base/src/org/netbeans/modules/java/source/save/Reformatter.java

```diff
@@ -2809,6 +2815,41 @@ private Boolean scanBindingPattern(Tree node, Void p) {
             return true;
         }
 
+        private Boolean scanGuardedPattern(Tree node, Void p) {
+            try {
+                Tree gpt = TreeShims.getGuardedPattern(node);
+                if (gpt != null) {
+                    Name name = TreeShims.getBinding(gpt);
```

Comment:
I am not sure if this works - the nested pattern in the guarded pattern can be parenthesized pattern. Probably should just call `scan` on `gpt`? (Which can't be null?)

Reply:
Addressed code review comment. Added code to handle  ---- case (String s) && s.<any> -> stmt;